templates/docs/examples/patterns/forms/checkbox-indeterminate.html

```diff
@@ -0,0 +1,17 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Forms / Checkbox indeterminate{% endblock %}
```

Comment:
This is not limited to `p-checkbox` pattern, so we should move the file to /examples/base/forms instead.

Reply:
Cool - that's done.